python/README.md

```diff
@@ -1251,7 +1251,33 @@ deactivate
 ```
 If you want to re-enter the virtualenv just follow the same instructions above about activating a virtualenv. There’s no need to re-create the virtualenv.
 
-##References
+### VirtualenvWrapper
+Even tough is not mandatory, there is a wrapper that sets some defaults about virtualenv use.
```

Comment:
Suggestion: maybe would be good put more details about what is VirtualenvWrapper. Maybe you can put the same description of the library:
```
virtualenvwrapper is a set of extensions to virtualenv tool. The extensions include wrappers for creating and deleting virtual environments and otherwise managing your development workflow, making it easier to work on more than one project at a time without introducing conflicts in their dependencies.

Features
Organizes all of your virtual environments in one place.
Wrappers for managing your virtual environments (create, delete, copy).
Use a single command to switch between environments.
Tab completion for commands that take a virtual environment as argument.
```